docs/validators/key-management/multikey-nodes.md

```diff
@@ -322,16 +322,219 @@ We strongly suggest to practice this process first on the public testnet. You sh
-4. switch off your single-key backup nodes (if you previously had ones);
-5. create a new multikey group, configure it as main group and let it sync. **Do not provide the `allValidatorsKeys.pem` keys yet!**. Go the next step **after all these nodes are synced**; 
-6. after the main group nodes are synced, copy the `allValidatorsKeys.pem` file to all nodes from the main group, switch off the main single-key nodes and restart the multikey nodes from the main group, so they will load the `allValidatorsKeys.pem` file;
-7. closely monitor all your nodes in the explorer, should be online and with their rating status increasing/at 100%. Repeat this step for a few times at 10 minutes interval. 
+1. Create your `allValidatorsKeys.pem` by manually (or through a text tool) concatenate all your `validatorKey.pem` files;
+2. Start a multikey group, **configure it as a backup group**, provide the `allValidatorsKeys.pem` file to all the nodes forming the group;
+3. Let this backup multikey group nodes sync and go the next step **after all these nodes are synced**;
+4. Switch off your single-key backup nodes (if you previously had ones);
+5. Create a new multikey group, configure it as main group and let it sync. **Do not provide the `allValidatorsKeys.pem` keys yet!**. Go the next step **after all these nodes are synced**; 
+6. After the main group nodes are synced, copy the `allValidatorsKeys.pem` file to all nodes from the main group, switch off the main single-key nodes and restart the multikey nodes from the main group, so they will load the `allValidatorsKeys.pem` file;
+7. Closely monitor all your nodes in the explorer, should be online and with their rating status increasing/at 100%. Repeat this step for a few times at 10 minutes interval. 
 
 Make sure that all operations from step 6 are made as quickly as possible. In case this step takes a long time, the backup multikey group should take over.
 
 :::caution
 Always attempt this process while closely monitor your nodes. If done correctly, your nodes might experience a brief rating drop (until the backup group takes over - if necessary)
 :::
+
+### Setup a multikey node
+
+**1. Login as root & update/upgrade the machine**
+
+```
+bash
+apt-get update
+apt-get upgrade
+apt autoremove
+```
+
+**2. Add and configure the ubuntu user**
+
+```
+adduser ubuntu
+```
+
+Set a long password
+
+```
+usermod -aG sudo ubuntu
+echo 'StrictHostKeyChecking=no' >> /etc/ssh/ssh_config
+visudo
+```   
+
+Add this line:
+
+```
+ubuntu  ALL=(ALL) NOPASSWD:ALL
+```
+
+Save & exit
+
+```
+sudo su ubuntu
+sudo visudo -f /etc/sudoers.d/myOverrides
+```
+
+Add this line:
+
+```
+ubuntu  ALL=(ALL) NOPASSWD:ALL
+```
+
+Save & exit
+
+**3. Configure ssh service:**
+
+```
+cd
+mkdir .ssh && chmod 700 .ssh && cd .ssh/
+nano authorized_keys
+```
+
+Paste your pubkey & save & exit
+The pubkey can be obtained by typing `cat ~/.ssh/id_rsa.pub` (or the name of your key) 
```

Comment:
... (or the name of your key) on the machine that will be used to connect to this host

Reply:
done